### PR TITLE
feat: Update spec's Tensor

### PR DESCRIPTION
**Pull request by simon-cj**
**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @your-reviewer

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: docs/spec-v1alpha1.md [hunk 1]
```diff
@@ -41,6 +41,14 @@ Tensors are multi-dimensional arrays with a uniform type.
 
     Size of the tensor.
 
+- **optype** string, OPTIONAL
+
+    OpType of the tensor, it is specical for PMML model.
+
+- **values** object, OPTIONAL
+
+    Values of the tensor, it is specical for PMML model.
```
Comment (gaocegege) [inline docs/spec-v1alpha1.md:50]:
```suggestion
    Values of the tensor, It is only used for PMML.
```

File: docs/spec-v1alpha1.md [hunk 2]
```diff
@@ -41,6 +41,14 @@ Tensors are multi-dimensional arrays with a uniform type.
 
     Size of the tensor.
 
+- **optype** string, OPTIONAL
+
+    OpType of the tensor, it is specical for PMML model.
```
Comment (gaocegege) [inline docs/spec-v1alpha1.md:46]: Same here

Comment (simon-cj) [reply docs/spec-v1alpha1.md]: comments are addressed, PTAL

**Pull request merged**